src/expressions/dataTypes/valueTypes/DayTimeDuration.ts

```diff
@@ -59,13 +59,23 @@ class DayTimeDuration extends AbstractDuration {
 		const hours = Math.abs(this.getHours());
 		const minutes = Math.abs(this.getMinutes());
 		const seconds = Math.abs(this.getSeconds());
-		const stringValue =
-			`${days ? `${days}DT` : 'T'}` +
-			`${hours ? `${hours}H` : ''}` +
-			`${minutes ? `${minutes}M` : ''}` +
-			`${seconds ? `${seconds}S` : ''}`;
+		const dayPart = `${days ? `${days}D` : ''}`;
+		const timePart =
+			(hours ? `${hours}H` : '') +
+			(minutes ? `${minutes}M` : '') +
```

Comment:
If I have the time 15:00:05 than won't that become 15H05S?
Wouldn't it make more sense to also include the minutes in that case or it defined like that in the spec?

Reply:
Correct, as spec.